index.html

```diff
@@ -92,151 +96,11 @@ <h2 class="name piesie">I'm Piesie Glad to see you!</h2>
-              class="size dot"
-            />
-            <h4 class="size date">2015</h4>
-          </div>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="languages">
-            <li class="html">html</li>
-            <li class="css">css</li>
-            <li class="javascript">javascript</li>
-          </ul>
-          <button>See Project</button>
-        </div>
-      </section>
+    <section class="works" id="portfolio"></section>
 
-      <section class="cards">
-        <img
-          src="images/Snapshoot-Portfolio-(1)-resize.png"
-          class="cards-pic"
-          alt="Portfolio third card"
-        />
-        <div>
-          <h1 class="section-title">Tonic</h1>
-          <div class="section-about">
-            <a href="#" class="size">CANOPY</a>
-            <img
-              src="images/Counter.png"
-              alt="Illustration of a dot 5"
-              class="size dot"
-            />
-            <h4 class="size back-end">Back End Dev</h4>
-            <img
-              src="images/Counter.png"
-              alt="Illustration of a dot 6"
-              class="size dot"
-            />
-            <h4 class="size date">2015</h4>
-          </div>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="languages">
-            <li class="html">html</li>
-            <li class="css">css</li>
-            <li class="javascript">javascript</li>
-          </ul>
-          <button>See Project</button>
-        </div>
-      </section>
-
-      <section class="cards">
-        <img
-          src="images/Snapshoot-Portfolio-resize1.png"
-          class="cards-pic"
-          alt="Portfolio fourth card"
-        />
-        <div class="cards-reverse">
-          <h1 class="section-title">Multi-Post Stories</h1>
-          <div class="section-about">
-            <a href="#" class="size">CANOPY</a>
-            <img
-              src="images/Counter.png"
-              alt="Illustration of a dot 7"
-              class="size dot"
-            />
-            <h4 class="size back-end">Back End Dev</h4>
-            <img
-              src="images/Counter.png"
-              alt="Illustration of a dot 8"
-              class="size dot"
-            />
-            <h4 class="size date">2015</h4>
-          </div>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="languages">
-            <li class="html">html</li>
-            <li class="css">css</li>
-            <li class="javascript">javascript</li>
-          </ul>
-          <button>See Project</button>
-        </div>
-      </section>
-    </section>
+    <div class="bg-popup">
+      <div class="popup"></div>
+    </div>
```

Comment:
- [ ] Kindly on the mobile version, I can't see the close button to close the window, Kindly fix it to match the original design.

![image](https://user-images.githubusercontent.com/35535130/173831104-6ae36f17-3efc-4071-8f33-822a3ed0d2e9.png)

Reply:
please kindly check the browser you are using

Reply:
Okay, lemme do that please